lessons/Early-Math/identity-property/langs/en-us/index.adoc

```diff
@@ -167,33 +167,104 @@ Students explore two functions - `scale` and `rotate` - each of which consumes a
 
 === Launch
 
-Let's say we have a number, `w`. The Identity Property tells us that we can multiply or divide `w` by 1... and we'll always get `w` back. If we multiply `w` by some value *other* than 1, however, we should expect a new outcome!
+You’ve discovered that multiplying or dividing an expression by 1 does not change its value. Similarly, adding or subtracting 0 results in the original value. Each of these four applications of the Identity Property is represented in the four Circles of Evaluation below.
+
+@ifnotslide{
+[.embedded, cols="^.^6a,^.^6a,^.^6a,^.^6a",stripes="none"]
+|===
+
+| @show{(coe '(* m 1))}
+| @show{(coe '(/ m 1))}
+| @show{(coe '(+ m 0))}
+| @show{(coe '(- m 0))}
+
+|===
+}
+
+Each of these four Circles evaluates to @math{m} which means *the Identity Property requires an operation that does… nothing!*
 
 @slidebreak
 
-`scale` is a @proglang function that can grow or shrink any image by a factor that we provide. Here is its contract:
+We can also represent these four applications of the Identity Property with the Circles of Evaluation below, with some made-up functions that take in just ONE input. The functions' describe what they do!
 
-@show{(contract "scale" '(("factor" "Number") ("img" "Image")) "Image")}
+@teacher{If students struggle with this idea, revisiting the idea of a "function machine" can be helpful!}
+
+@ifnotslide{
+[.embedded, cols="^.^6a,^.^6a,^.^6a,^.^6a",stripes="none"]
+|===
 
-Let's say we enter @show{(code '(scale 3 (triangle 30 "solid" "purple")))}. We can expect @proglang to produce a 90-pixel solid purple triangle. The original image's size was _tripled_.
+| @show{(coe '(multiply-by-one m))}
+| @show{(coe '(divide-by-one m))}
+| @show{(coe '(add-zero m))}
+| @show{(coe '(subtract-zero m))}
 
-@lesson-instruction{
-Use @starter-file{scale-rotate} to experiment with the `scale` function and complete @printable-exercise{scale-rotate-identity.adoc} through question 5.
+|===
 }
 
 
+No matter what value we use for @math{m}, we will get that value back.
+
+@slidebreak
+
+
+@lesson-roleplay{
+Select four students - one to _act out_ each of the four functions above (`multiply-by-one`, `divide-by-one`, `add-zero`, and `subtract-zero`). Make it clear to the class what each function's name is. Emphasize that each function expects a Number, and will produce Number.
```

Comment:
I love this idea, but I worry that it will get boring too quickly. Could you add another twist, which is that the class gets to brainstorm _other_ "actors" who follow the identity property? This would be a nice callback to some of your "tricky" examples in the prior section, like "a * (8 - 8)".